server/plugins/resolver.js

```diff
@@ -0,0 +1 @@
+module.exports = require.resolve
```

Comment:
I wonder if it’s worth it to put this into it’s own file? I think we use `require.resolve` directly in other modules.

Also, all files in `server/plugins/` are valid hapi plugins, while this exposes an API. It’s just a little nitpick, I’d merge it in as is, we can also refactor it later :) But please let me know what your thinking is behind this file